core/opds2.py

```diff
@@ -192,8 +201,6 @@ def feed_metadata(self):
         return {
             "title": self.title,
             "itemsPerPage": self.pagination.size,
-            "currentPage": math.ceil(self.pagination.offset / self.pagination.size)
```

Comment:
Do you know is someone or something using/expecting this `currentPage`?

Reply:
I'm not sure anybody uses this right now, it is a relatively new feature. James English can answer this question though.
The OPDS2 spec defines it as a `MAY` attribute.